content/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax.md

```diff
@@ -24,6 +24,11 @@ To create a heading, add one to six <kbd>#</kbd> symbols before your heading tex
 
 ![Rendered H1, H2, and H6 headings](/assets/images/help/writing/headings-rendered.png)
 
+The use of two or more headings automatically generates a table of contents within the header. Each respective heading title will be listed in the table of contents and navigate within the document to the selected section. 
+
+![Headings Table of Contents](https://i2.wp.com/user-images.githubusercontent.com/7900087/113821370-df915480-9730-11eb-8aed-bdc50e2212d5.gif?ssl=1)
```

Comment:
This image isn't showing up in the [preview](https://docs-15218--patch-2.herokuapp.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#headings). We're also trying stop avoid using animated GIFs in the help as they are bad for accessibility. 

Can I suggest that you add the image below to the `images` directory [here](https://github.com/wonntann/docs/upload/patch-2/assets/images/help/repository) as `table-of-contents.png`? Then we can update this to:

```suggestion
![Screenshot highlighting the table of contents icon](/assets/images/help/repository/table-of-contents.png)
```

<img width="889" alt="table-of-contents" src="https://user-images.githubusercontent.com/1877141/153183249-67e2811d-6d48-4647-b3cc-37bde91c5b4f.png">

Reply:
Ok, sounds good. I committed a png image for the toc. I overlooked the accessibility, if you need any assistance with increasing accessibility please let me know. Thanks!